.sass-lint.yml

```diff
@@ -0,0 +1,103 @@
+#########################
+## Sample Sass Lint File
+#########################
+# Linter Options
+options:
+  # Set the formatter to 'html'
+  formatter: stylish
+# File Options
+files:
+  include: 'src/**/*.scss'
+# Rule Configuration
+rules:
+  # Extends
+  extends-before-mixins: 2
+  extends-before-declarations: 2
+  placeholder-in-extend: 2
+
+  # Mixins
+  mixins-before-declarations:
+    - 2
+    -
+      exclude:
+        - 'respond-at-and-above'
+        - 'respond-between'
+        - 'respond-below'
+        - 'after'
+        - 'before'
+
+  # Line Spacing
+  one-declaration-per-line: 2
+  empty-line-between-blocks: 2
+  single-line-per-selector: 2
+
+  # Disallows
+  no-color-keywords: 2
+  no-color-literals: 0
+  no-css-comments: 0
+  no-debug: 2
+  no-duplicate-properties: 2
+  no-empty-rulesets: 2
+  no-extends: 0
+  no-ids: 2
+  no-important: 2
+  no-invalid-hex: 2
+  no-mergeable-selectors:
+    - 2
+    -
+      whitelist:
+        - 'from'
+        - 'to'
+  no-misspelled-properties: 2
+  no-qualifying-elements: 2
+  no-trailing-zero: 2
+  no-transition-all: 2
+  no-url-protocols: 0
+  no-vendor-prefixes: 2
```

Comment:
I think it's overly complex to add a special prefixing module, and it would obfuscate the knowledge that it's a non-standard, vendor-only property. We do currently use that rule in Casper Rails, among others (`-moz` textfield ones, off the top of my head).

IMO, all of us are on the same page that autoprefixer handles prefixing, unless the property is non-standard. In those cases I want to see that we're using a non-standard property, so I would be in favor of taking this one out.